src/pkg/services/m365/m365.go

```diff
@@ -209,7 +210,7 @@ func GetUserInfo(
 	acct account.Account,
 	userID string,
 ) (*api.UserInfo, error) {
-	ac, err := makeAC(acct)
+	ac, err := makeAC(ctx, acct, path.ExchangeService)
```

Comment:
Sanity check: Is `path.ExchangeService` correct here?

Reply:
`GetInfo` calls `GetMailbox` which works under exchange concurrency constraints.